src/session/sidebar/chat_row.rs

```diff
@@ -358,6 +359,43 @@ fn stringify_message(message: Message) -> String {
                 format!(", {}", dim_and_escape(&data.caption.text))
             }
         ),
+        MessageContent::MessageCall(data) => {
+            match data.discard_reason {
+                CallDiscardReason::Declined => {
+                    if message.is_outgoing() {
+                        // Telegram Desktop/Android labels declined outgoing calls just as
+                        // "Outgoing call" and puts a red arrow in the message bubble. We should be
+                        // more accurate here.
+                        if data.is_video {
+                            gettext("Unanswered video call")
+                        } else {
+                            gettext("Unanswered call")
+                        }
+                    // Telegram Android labels declined incoming calls as "Incoming call". Telegram
+                    // Desktop labels it as "Declined call" and is a bit inconsistent with outgoing
+                    // calls ^.
+                    } else if data.is_video {
+                        gettext("Declined video call")
+                    } else {
+                        gettext("Declined call")
+                    }
+                }
+                CallDiscardReason::Disconnected
+                | CallDiscardReason::HungUp
+                // TODO: How can this be reproduced? What does this mean? For now we would just
```

Comment:
The documentation says `The call wasn't discarded, or the reason is unknown`. But it seems to be used more as a [general unknown reason](https://github.com/UnigramDev/Unigram/blob/53f0d5bf988ee40ba61eea8a807b2eca721436a3/Unigram/Unigram/Views/VoIPPage.xaml.cs#L710) for a call to end, so I guess this is correct.